scripts/doc/Makefile

```diff
@@ -228,7 +228,6 @@ install:
 	# -b gh-pages fetches only the branch for the gh-pages
 	git clone -b gh-pages --single-branch --no-checkout --depth 1 https://github.com/mne-tools/mne-biomag-group-demo _build/mne-biomag-group-demo.github.io
 	touch _build/mne-biomag-group-demo.github.io/.nojekyll
-	make html
```

Comment:
@jasmainak this is a process change to do `make clean; make html; make install` instead of just `make install` because you can iterate on `make html` (or for me, `make html-noxvfb`) without needing to re-run generation once you get one you like

Reply:
okay, I just noticed the nice refactor of the `Makefile`. That looks great to me. Thanks a lot!